charts/vela-core/templates/defwithtemplate/webservice.yaml

```diff
@@ -64,15 +64,15 @@ spec:
       	}
       }
       parameter: {
-      	// +usage=specify app image
+      	// +usage=Which image would you like to use for your service
```

Comment:
The asking-question way is suitable for appfile, but it might sound a little bit of strange in Cli `vela svc deploy -t webservice -h` or `vela autoscale -h`.

Reply:
I think it's OK, mainly focus on appfile